README.md

```diff
@@ -28,6 +28,9 @@ http://entrproject.org/) utility:
 ```bash
 find . -name "*_shpec.sh" | entr shpec
 ```
+__Warning__: If you use `zsh` you should disable the `end` keyword. (to do that, run `disable -r end`.
```

Comment:
On 05/18/15 06:01, Adriean Khisbe wrote:
> In README.md <https://github.com/rylnd/shpec/pull/86#discussion_r30480577>:
>
>> @@ -28,6 +28,9 @@ http://entrproject.org/) utility:
>>  ```bash
>>  find . -name "*_shpec.sh" | entr shpec
>>  ```
>> +__Warning__: If you use `zsh` you should disable the `end` keyword. (to do that, run `disable -r end`.
>
> I really thing it belongs to the README, but indeed there it's noise.
> (I'm not sure most user of zsh now about disabling keyword. (I discovered this very recently))
>
> Maybe we should do a section /Shell Specificity/, where we put bash, zsh, ksh, or sh related nnotes.

I could use a couple of notes on the known issues with `zsh`.

I had to find the look up the *zsh reserved words* to see `foreach ... end`  (Which looks like a csh-ism)

This information would fit best in a COMPATIBILITY or NOTES document, besides the README.